Source/WebKit/Shared/WebCoreArgumentCoders.h

```diff
@@ -76,6 +77,12 @@ template<> struct ArgumentCoder<sk_sp<SkColorSpace>> {
     static void encode(StreamConnectionEncoder&, const sk_sp<SkColorSpace>&);
     static std::optional<sk_sp<SkColorSpace>> decode(Decoder&);
 };
+
+template<> struct ArgumentCoder<sk_sp<SkData>> {
```

Comment:
This file should be decreasing until it is removed.  *.serialization.in has "wrapped by" to accomplish this.  It may need a little adapting.

Reply:
I am with you on this goal but that will require some surgery. Is it ok to land this and open a bug to make `wrapped by` work on non-cocoa situations?